siatest/renterhost/renterhost_test.go

```diff
@@ -0,0 +1,100 @@
+package renterhost
+
+import (
+	"bytes"
+	"testing"
+
+	"github.com/NebulousLabs/Sia/siatest"
+)
+
+// TestRenterHost executes a number of subtests using the same TestGroup to
+// save time on initialization
+func TestRenterHost(t *testing.T) {
```

Comment:
Overall test layout looks good, but I think fine to just change it to 'TestRenter'